IncludeToolbox/IncludeFormatter/IncludeLineInfo.cs

```diff
@@ -92,6 +92,8 @@ public static IncludeLineInfo[] ParseIncludes(string text, bool removeEmptyLines
                 if (multiLineCommentStart > -1 && multiLineCommentStart < includeOccurence) // Multi line comment started in same line, but before #include.
                     continue;
 
+                if (lines[line].Contains(".inl>") || lines[line].Contains(".inl\"")) // Don't remove inline files.
```

Comment:
No I don't think so, but the IncludeLineInfo is also used for other tools, namely sorting (which I do use quite a lot). The sorting however, can easily be configured by the user to account for inl files.

You have a good point for the .inc though! I think it is best if the user could configure a list of regexes for "never remove this" that we can check before we try to remove an include. I'll get right on it and put .inc, .inl and _inl.h on the list by default.